store/pruning.go

```diff
@@ -1,39 +1,33 @@
 package store
 
-import "fmt"
+import (
+	"fmt"
+)
 
 // PruneBlocks removes block up to (but not including) a height. It returns number of blocks pruned.
-func (s *DefaultStore) PruneBlocks(heightInt int64) (uint64, error) {
-	if heightInt <= 0 {
-		return 0, fmt.Errorf("height must be greater than 0")
+func (s *DefaultStore) PruneBlocks(from, to uint64) (uint64, error) {
+	if from <= 0 {
+		return 0, fmt.Errorf("from height must be greater than 0")
 	}
```

Comment:
seems like a usess branch since if we try to fetch h=0 it will just continue
https://github.com/dymensionxyz/dymint/blob/953a9b82456393d4e1ab5270719a02dbc68af528/store/pruning.go#L33